### DISCUSSION TITLE
C盘里的Partitions文件夹内容能删除吗？

### DISCUSSION THREAD

**Discussion by dongrentianyu** [tiddly-gittly/TidGi-Desktop#567] [Q&A] [answered]
我发现这个文件夹占了3个G，但里面的内容又打不开，不知道是啥。我的打开路径是"C:\Users\34656\AppData\Roaming\TidGi\Partitions"。

![image](https://github.com/tiddly-gittly/TidGi-Desktop/assets/109711900/6e25a41d-6a90-4880-b9fe-5f81adaaf497)

感觉像是缓存文件。不知道能不能删除。

**Comment by linonetwo** [comment, marked as answer]
应该是网页缓存，没用，可以删。只有 settings 还有 wiki 文件夹是重要的。

**Reply by dongrentianyu** [reply]
好
